README.md

```diff
@@ -1,127 +1,126 @@
 Docker open-oni
 ===============
 
-Run open-oni (community chronicling america fork) in Docker.
+(Production instructions TBD - for now this is just for doing open-oni dev)
```

Comment:
The docker installation instructions on open-oni's wiki have a description of setting up docker-machine / docker itself on mac, and then overlap a bit with the `dev.sh` info, though not in nearly so much detail.  Possibly put a link to the docker installation instructions at the top of this file, or talk about blowing away the wiki version rather than keeping instructions in two places?
https://github.com/open-oni/open-oni/wiki/Docker-Setup-OS-X